docs/js/features/odata/use-generated-client.md

```diff
@@ -405,9 +403,11 @@ The below example leverages OData v4 exclusive features to build a more complex
 (to change, might not work)
 
 ```ts
-//Fetch all business partners where:
-//- the last name is at least twice as long as the first name
-//- AND the combined string of first and last name does not contain 'bob'
+/*
+  Fetch all business partners where:
+    - the last name is at least twice as long as the first name
+    - AND the combined string of first and last name does not contain 'bob'
+*/
```

Comment:
How did you do that, the compiler gave me an error if I had a `/*comment*/`

Reply:
Good catch, I didn't even check that.